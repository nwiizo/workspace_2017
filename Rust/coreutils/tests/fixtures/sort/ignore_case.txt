aaa
ccc
eee
ggg
BBB
DDD
FFF
