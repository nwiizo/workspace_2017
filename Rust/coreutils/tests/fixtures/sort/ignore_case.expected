aaa
BBB
ccc
DDD
eee
FFF
ggg
